Adafruit_SSD1306.cpp

```diff
@@ -452,13 +452,17 @@ void Adafruit_SSD1306::ssd1306_command(uint8_t c) {
             platforms where a nonstandard begin() function is available
             (e.g. a TwoWire interface on non-default pins, as can be done
             on the ESP8266 and perhaps others).
+    @param  flip
+            If true, flips the display by 180 degrees. This allows flexibility
```

Comment:
I believe it would be would be more accurate to say "flips the display horizontally" rather than "flips the the display by 180 degrees", since it's not a rotation, and doesn't affect the vertical axis. Also maybe change "flip" to something like "flipHorizontal" or "flipX" to clarify that.

Reply:
You're right, I'll change the parameter name to `flipVertical`, and update the comment/`@param` description.

> I'm actually a little confused about whether the "flip" is vertical or horizontal here. Your commit comments say "along the X axis" but then you say in the description here that you want to support "upside-down" drawing.

Yes, I think I confused myself between the X and Y axes. It is meant to be along the Y axis (so it allows rotating the display 180 degrees). Mybad!

I didn't realise there was a GFX function. Thanks for pointing that out! I haven't explored using it, but looking at GFX's `setRotation` function, it only swaps the width and height around. So my 128x32 display becomes 32x128 **in software** but remains 128x32 physically (in theory, and from what I'm reading). The function itself is not used within the SSD1306 library, but if that function is prefered, it might be worth investigating.

I chose to put it here because I had found an [Adafruit Forum topic](https://forums.adafruit.com/viewtopic.php?t=25438#p130858) on flipping the display. That suggests sending commands via the I2C bus. I tried that and that worked flawlessly, so it seemed like a good avenue to go down. Was I wrong to do so? Would it be best to use GFX's `setRotation` function?

I'm updating the PR description with some images to illustrate what the change does vidually.

Reply:
@BillyDonahue I've looked into the GFX function `setRotation`, and it seems that actually works better for rotating the display. Thank you for bringing it up! I'll close this PR, as the scope was already met previously. 

And there I thought I was making a contribution! Lesson learnt: read **all** the code, including parent classes and dependencies!